src/index.js

```diff
@@ -148,7 +152,12 @@ const printers = {
               return childValue ? [...accumulator, ...childValue] : accumulator;
             }, []),
             ...map.children.nodes.reduce((accumulator, childNode) => {
-              const childValue = node[childNode];
+              let childValue = node[childNode];
+              if (node.kind === "include") {
+                while (childValue && childValue.kind === "parenthesis") {
```

Comment:
this seems fishy to me - why not just add `parenthesis` to the map above?

Reply:
@mgrip `parenthesis` can have other `parenthesis` and they also have `parenthesis`

Reply:
right... but as long as `parenthesis` is in the map once it should work - don't think it matters that its recursive

Reply:
@mgrip i get error about comments not printed :confused:

Reply:
hrm I'm trying to check out your branch to see but I can't seem to find it...

Reply:
anyways just tried it myself and it worked

```javascript
{
  kinds: ["parenthesis"],
  children: { listNodes: [], nodes: ["inner"] }
},
{
  kinds: ["include"],
  children: { listNodes: [], nodes: ["target"] }
}
```

Reply:
@mgrip your solution is better :+1: I will do this in near future :+1: